src/object_store.cpp

```diff
@@ -50,19 +50,24 @@ const size_t c_primaryKeyObjectClassColumnIndex =  0;
 const char * const c_primaryKeyPropertyNameColumnName = "pk_property";
 const size_t c_primaryKeyPropertyNameColumnIndex =  1;
 
+constexpr const char* result_sets_type_name = "class___ResultSets";
```

Comment:
It's not clear why any of the changes to this file or object_store.hpp are needed. The partial sync code already handles creating the ResultsSet table.